pydis_site/apps/content/README.md

```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
+relatively little code in there, most of it is concerned with serving our
+content.
+
+The markdown files hosting our content can be found in the
+[`resources/`](./resources) directory, and contain
+
+
+## Contributing pages
+
+Contributing pages is covered extensively in our online guide, which you can
+find
+[here](https://www.pythondiscord.com/pages/guides/pydis-guides/how-to-contribute-a-page/)
+or alternatively read it directly at
+[`resources/guides/pydis-guides/how-to-contribute-a-page.md`](./resources/guides/pydis-guides/how-to-contribute-a-page.md).
+
+
+## Directory structure
+
+Let's look at the structure in here:
+
+- `resources` contains the static Markdown files that make up our site's
+  [pages](https://www.pythondiscord.com/pages/)
+
+- `migrations` contains standard Django migrations. As the `content` app
+  contains purely static markdown files, no migrations are present here.
```

Comment:
Why do we even keep this directory around then? Will Django complain if it can't find it?

Reply:
Good point. I've removed it in 4ad92b98a0df047b12e302ce8d849c2bd85862f1.